documentation/modules/ROOT/pages/integration-with-aap.adoc

```diff
@@ -0,0 +1,176 @@
+= Integration with Ansible Automation Platform (AAP)
+include::_attributes.adoc[]
+:profile: 5g-ran-lab
+
+
+[#aap-overview]
+== AAP Overview
+
+link:https://www.redhat.com/en/technologies/management/ansible[Red Hat Ansible Automation Platform] allows for the configuration and management of systems at scale through simple IT automation. If unfamiliar with AAP, see the link:https://docs.ansible.com/ansible/latest/index.html[AAP Docs]. In this section, we will discuss the need for AAP integration with 5G RAN deployments on OpenShift.
```

Comment:
Change the link from community docs to Red Hat docs. i.e: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4